test/battle/move_effect/chilly_reception.c

```diff
@@ -0,0 +1,79 @@
+#include "global.h"
+#include "test/battle.h"
+
+ASSUMPTIONS
+{
+    ASSUME(gBattleMoves[MOVE_CHILLY_RECEPTION].effect == EFFECT_CHILLY_RECEPTION);
+}
+
+SINGLE_BATTLE_TEST("Chilly Reception switches the user out")
+{
+    GIVEN {
+        PLAYER(SPECIES_SLOWKING_GALARIAN);
+        PLAYER(SPECIES_SLOWPOKE_GALARIAN);
+        OPPONENT(SPECIES_WOBBUFFET);
+    } WHEN {
+        TURN { MOVE(player, MOVE_CHILLY_RECEPTION); SEND_OUT(player, 1); }
+    } SCENE {
+        MESSAGE("Slowking is preparing to tell a chillingly bad joke!");
+        ANIMATION(ANIM_TYPE_MOVE, MOVE_CHILLY_RECEPTION, player);
+        MESSAGE("It started to snow!");
```

Comment:
Please add the `{pokemon} went back to {player}` message.

Reply:
You need 2 spaces between to and 1, `to  1`

Reply:
D'oh! I see it now! Thank you!!